setup.py

```diff
@@ -49,12 +49,12 @@ def get_version(*file_paths):
     ],
     include_package_data=True,
     install_requires=[
-        "Django>=1.8.15,<1.10",
-        "djangorestframework>=3.2.3,<3.3.0",
-        "django-config-models",
+        "Django>=1.8.15,<1.11",
+        "djangorestframework>=3.2.3,<3.7.0",
+        "django-config-models==0.1.6",
```

Comment:
@snowcrshd nit: I'd suggest >=0.1.6, but this can be addressed later when (and if) next DCM version comes out.